QUICKSTART.md

```diff
@@ -1,6 +1,5 @@
-
 <!---
-Copyright 2020 Crown Copyright
+Copyright 2018-2021 Crown Copyright
```

Comment:
Had a look and there are about 1/2 dozen with only the 2020.  All are md files.

Reply:
Won't do as out-of-scope.
Better addressed when the documentation of the repo is updated in a future ticket